docs/content/concepts/ops-jobs-graphs/jobs.mdx

```diff
@@ -0,0 +1,262 @@
+To do this, you first define a graph with the <PyObject object="graph" decorator /> decorator.
+
+```python file=/concepts/ops_jobs_graphs/jobs_from_graphs.py startafter=start_define_graph endbefore=end_define_graph
+from dagster import graph, op
+
+
+@op(required_resource_keys={"server"})
+def interact_with_server(context):
+    context.resources.server.ping_server()
+
+
+@graph
+def do_stuff():
+    interact_with_server()
+```
+
+Then you build jobs from it using the <PyObject object="GraphDefinition" method="to_job" /> method:
+
+```python file=/concepts/ops_jobs_graphs/jobs_from_graphs.py startafter=start_define_jobs endbefore=end_define_jobs
+from dagster import ResourceDefinition
+
+prod_server = ResourceDefinition.mock_resource()
+local_server = ResourceDefinition.mock_resource()
+
+prod_job = do_stuff.to_job(resource_defs={"server": prod_server}, name="do_stuff_prod")
+local_job = do_stuff.to_job(
+    resource_defs={"server": local_server}, name="do_stuff_local"
+)
+```
+
+`to_job` accepts the same arguments as the <PyObject object="job" decorator /> decorator: you can provide [resources](/concepts/resources), [configuration](/concepts/configuration/config-schema), [hooks](/concepts/ops-jobs-graphs/op-hooks), tags, and an [executor](/deployment/executors).
+
+---
+
+## Configuring jobs
+
+Ops and resources often accept [configuration](/concepts/configuration/config-schema) that determines how they behave. By default, you supply configuration for these ops and resources at the time you launch the job.
+
+When constructing a job, you can customize how that configuration will be satisfied, by passing a value to the `config` parameter of the <PyObject object="GraphDefinition" method="to_job" /> method or the <PyObject object="job" decorator /> decorator. The options are discussed below:
+
+- [Hardcoded configuration](#hardcoded-configuration)
+- [Partitioned configuration](#partitioned-configuration)
+- [Config mapping](#config-mapping)
+
+### Hardcoded configuration
+
+You can supply a config dictionary. The supplied dictionary will be used to configure the job whenever the job is launched. It will show up in the Dagit Launchpad and can be overridden.
+
+```python file=/concepts/ops_jobs_graphs/jobs_with_default_config.py
+from dagster import job, op
+
+
+@op(config_schema={"config_param": str})
+def do_something(context):
+    context.log.info("config_param: " + context.op_config["config_param"])
+
+
+default_config = {"ops": {"do_something": {"config": {"config_param": "stuff"}}}}
+
+
+@job(config=default_config)
+def do_it_all_with_default_config():
+    do_something()
+
+
+if __name__ == "__main__":
+    # Will log "config_param: stuff"
+    do_it_all_with_default_config.execute_in_process()
+```
+
+### Partitioned jobs
+
+You can supply a <PyObject object="PartitionedConfig" />. It defines a discrete set of "partitions", along with a function for generating config for a partition. You can configure a job run by selecting a partition.
```

Comment:
Maybe worth mentioning that this works differently for software-defined assets.

Reply:
I feel like that info *should* be on the Partitions page, not here.

Reply:
I agree that the main documentation for this should be on the Partitions page, but I think the paragraph here is misleading, because it makes a statement that sounds like it applies to all jobs, but which only applies to op-based jobs.  I.e. you can't supply a PartitionedConfig to a partitioned job.

Reply:
That's fair!